docs/UsingNavigators.md

```diff
@@ -102,11 +102,14 @@ navigator.pop();
 
 A more complete example that demonstrates the pushing and popping of routes could therefore look something like this:
```

Comment:
I am not too happy with the redundant use of colons here, once L105 is added in this PR.

I suggest the following:

" A more complete example that demonstrates the pushing and popping of routes could therefore look like the following. Change your main file to:"

I am not 100% happy with the use of "main file" here as it can confuse a new user, though I don't have a better suggestion. The concern is that a new user would not necessarily know if this refers to the main App.js file (incorrect), or the actual file for the SimpleNavigationComponent we're working with in this example (correct).